### DISCUSSION TITLE
Future talent ready

### DISCUSSION THREAD

**Discussion by mnigam504** [github/india#394] [Students]
I applied through AICTE website and there were no follow up instruction. So today I registered at FTR website and read FAQ and started the program.
I have filled up all the orientation link. Now self learning tab is open. What should I do next?
How to complete the certification.
Also is there any introduction video for this program and orientation session?

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any `Official FRT Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Currently, we are helping all the `Future Ready Talent` Interns clear their doubts & queries on a personal basis either on chat or voice, whichever they prefer. We will also be assisting them on Projects & help them complete this internship 🤗

Link to join: https://discord.gg/drQ7ySCdX2
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏
